CHANGELOG.md

```diff
@@ -2,16 +2,38 @@
 
 ## [Unreleased]
 
+## [1.3.0] - 2022-04-22
+
 ### General
 
 - Add support information into versioning specification.
+- Add behaviors specification.
 - Refactor profiling into existing specification structure.
 
 #### Bug fixes
 
 - Replaced the example for Java system properties in `specification/configuration.md`.
 - Increase profiling rate from 1s to 10s.
 
+### Configuration
+
+- Change the Kubernetes package management solutions configuration option
+  default for `splunkPlatform.metricsEnabled` to be `false`.
+- Add `SPLUNK_REALM` to required environment variables that need to be
+  supported by instrumentation libraries.
+- Add `SPLUNK_PROFILER*` environment variables
+  (`SPLUNK_PROFILER_CALL_STACK_INTERVAL`, `SPLUNK_PROFILER_ENABLED`,
+  `SPLUNK_PROFILER_LOGS_ENDPOINT`) to the required configuration for
+  instrumentation libraries.
+- Add `OTEL_TRACES_EXPORTER` to required environment variables that need to be
+  supported by serverless instrumentation libraries.
+
+### Semantic Conventions
+
+- Correct stability guarantee of the RUM section as experimental.
+- Use `os.name` instead of `os.type` for mobile RUM libraries.
+- Add experimental profiling semantic conventions.
```

Comment:
```suggestion
### Configuration

#### Breaking Changes

- Change the Kubernetes package management solutions configuration option
  default for `splunkPlatform.metricsEnabled` to be `false`.

#### Enhancements

- Add `SPLUNK_REALM` to required environment variables that need to be
  supported by instrumentation libraries.
- Add `SPLUNK_PROFILER*` environment variables
  (`SPLUNK_PROFILER_CALL_STACK_INTERVAL`, `SPLUNK_PROFILER_ENABLED`,
  `SPLUNK_PROFILER_LOGS_ENDPOINT`) to the required configuration for
  instrumentation libraries.
- Add `OTEL_TRACES_EXPORTER` to required environment variables that need to be
  supported by serverless instrumentation libraries.

### Semantic Conventions

#### Breaking Changes

- Use `os.name` instead of `os.type` for mobile RUM libraries.

#### Bugfixes

- Correct stability guarantee of the RUM section as experimental.

#### Enhancements

- Add experimental profiling semantic conventions.
```